src/ankh/cmd/ankh/main.go

```diff
@@ -323,22 +329,29 @@ func inspect(ctx *ankh.ExecutionContext,
 	var result string
 
 	ankhFile, err := ankh.ParseAnkhFile(ctx.AnkhFilePath)
-	if err == nil {
-		log.Infof("- OK: %v", ctx.AnkhFilePath)
-	}
 	check(err)
 
-	if len(ankhFile.Charts) > 0 {
-		ctx.Logger.Debug("Inspecting charts")
-		for _, chart := range ankhFile.Charts {
-			output, err := cb(ctx, chart, ankhFile)
-			check(err)
-			result += output
-		}
-	} else {
+	if len(ankhFile.Charts) < 1 {
 		log.Infof(
 			"%s does not contain any charts. Inspect commands only operate on ankh.yaml files containing charts",
 			ctx.AnkhFilePath)
+		return
+	}
+
+	ctx.Logger.Debug("Inspecting charts")
+	for _, chart := range ankhFile.Charts {
+		if ctx.Chart != "" && chart.Name != ctx.Chart{
```

Comment:
run gofmt over this

Reply:
Was thinking about that, maybe that's a good command to add to make
like `make format` or something

Reply:
or maybe a pre-commit hook!